apis/v1/gateway_types.go

```diff
@@ -226,7 +226,6 @@ type GatewaySpec struct {
 	//
 	// Support: Core
```

Comment:
```suggestion
	// Support: Extended
```

I think we can theoretically call this "extended" because we have some conformance tests that will cover a subset of implementations. With that said, it's possible that a cloud-based solution may both support this feature and not pass those tests. That part starts to make it feel a bit closer to "implementation-specific", but will defer to you on which you think is more accurate here. In either case, I'm confident that we can't call this (or any of the fields within infra) "Core".

Reply:
Ah right, not sure what I was thinking here. Extended is what I was going for, with individual fields perhaps being implementation specific.